core/src/main/java/org/apache/gravitino/GravitinoEnv.java

```diff
@@ -220,6 +220,26 @@ public IdGenerator idGenerator() {
     return idGenerator;
   }
 
+  /**
+   * Get the CatalogManager associated with the Gravitino environment.
+   *
+   * @return The CatalogManager instance.
+   */
+  public CatalogManager catalogManager() {
+    Preconditions.checkNotNull(catalogManager, "GravitinoEnv is not initialized.");
+    return catalogManager;
+  }
+
+  /**
+   * Get the EventBus associated with the Gravitino environment.
+   *
+   * @return The EventBus instance.
+   */
+  public EventBus eventBus() {
+    Preconditions.checkNotNull(eventBus, "GravitinoEnv is not initialized.");
```

Comment:
I remembered that for Iceberg rest service, some modules will not enabled, so it will be null, can you please confirm this with @FANNG1 , also use `checkArgument` not `checkNotNull`.

Reply:
I think this doesn't seem to conflict with the fact that if the variable is not initialized it is not allowed to get it.

If the Iceberg REST service disables the modules, it does not need to call the method.

Reply:
`If the Iceberg REST service disables the modules, it does not need to call the method.` 
+1